railties/test/generators/app_generator_test.rb

```diff
@@ -901,8 +863,9 @@ def test_css_option_with_tailwind_uses_cssbundling_gem_when_using_node
 
   def test_css_option_with_asset_pipeline_sass
     run_generator_and_bundler [destination_root, "--css=sass"]
+
+    assert_includes @rails_commands, "dartsass:install", "`dartsass:install` expected to be called, but wasn't."
```

Comment:
Same comment here regarding test coverage in the `dartsass-rails` gem and changing `run_generator_and_bundler` to `run_generator`.

Reply:
Fixed by rails/dartsass-rails#49